modules/data/db_manager.py

```diff
@@ -40,7 +42,9 @@ def get_db(cls) -> Tuple[sqlite3.Connection, sqlite3.Cursor]:
         Returns:
             Tuple[sqlite3.Connection, sqlite3.Cursor]: sqlite database connection and cursor
         """
-        db_path = get_abs_path("data", "db", "sqlite.db")
+        db_path = get_abs_path(*cls.db_path)
+        if not os.path.exists(db_path):
+            open(db_path, 'w').close()
```

Comment:
what is this for? to create an empty file?

Reply:
Yep. On startup, the bot will create the tables, if missing. This just makes sure that the file is actually present. It can be a simple empty file

Reply:
we can even add it in the project to save some code here, but it's the same for me